src/ec/suite_b/ops.rs

```diff
@@ -985,6 +982,14 @@ mod tests {
         );
     }
 
+    #[test]
+    fn p521_point_mul_base_test() {
+        point_mul_base_tests(
+            &p521::PRIVATE_KEY_OPS,
+            test_file!("ops/p521_point_mul_base_tests.txt"),
```

Comment:
Is the problem possilby that the expected result is a Point, which is Montgomery-encoded, and you generated p521_point_mul_base_tests.txt for 64-bit and not for 32-bit?

Reply:
I changed it to have two separate files, although we may want to either change the file format, or alternatively use non-montgomery results, like boringssl does and convert the result instead.

Reply:
Seems to pass now